Introduce a Primer element (#242)
Lattice-Automation/seqviz: README.md

In the example above, the "Strong promoter" would span the first to twenty-second base pair.

#### `primers (=[])`

> An array of `Primer`s to render. Each `Primer` requires 0-based start (inclusive) and end (exclusive) indexes. `name`s are rendered on top of the annotations. Set the primer's direction to `1` for forward primer and `-1` for reverse primer.

Reviewer: ```suggestion
An array of `Primer`s to render. Each `Primer` requires 0-based start (inclusive) and end (exclusive) indexes. `name`s are rendered on top of the primers. Set the primer's direction to `1` for forward primer and `-1` for reverse primer.
```

```js
primers = [
  { start: 33, end: 53, name: "LacZ Foward Primer", direction: 1 },

>   { id: "18fbd562-9f87-4a17-aa20-b8448538cbe6", start: 3098, end: 3128, name: "LacZ Reverse Primer", direction: -1, color: "#FAA887" },

Reviewer: ```suggestion
  { start: 3098, end: 3128, name: "LacZ Reverse Primer", direction: -1, color: "#FAA887" },
```

The `id` thing is kind just for us internally. I realize that because we let folks render a `Linear` or `Circular` component directly, they are then _required_ to provide an id if they pass it to either of those components directly. But that can be a later fix imo (and we have the same issue rn w/ Annotations)

];
```

> In the example above, the foward and reverse primers of LacZ are define by the direction parameter. Notice that id and color could be optionally used.

Reviewer: ```suggestion
In the example above, the forward and reverse primers of LacZ are define by the direction parameter. Notice that color could be used optionally.
```
